verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -244,12 +248,6 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
                 <Paper>
                     <MaterialTable
                         columns={[
-                            {
```

Comment:
IIRC the last discussion we had was to keep the ID field in to give curators a way to name a case when discussing e.g. duplicates.

Reply:
we could show it in the details page (or in the details pane feature of this table that we're not usng yet) but here it takes some big real estate for no good reason imo, let's discuss with Shru.